resources/migrations/000_migrations.yaml

```diff
@@ -14228,6 +14228,14 @@ databaseChangeLog:
             constraintName: fk_sandboxes_ref_permissions
             nullable: true
 
+  - changeSet:
+      id: v46.00-076
```

Comment:
@noahmoss is already using this migration ID for something else -- don't forget to "claim" your migration IDs in the `#migrations` channel on Slack until we come up with a better system for this stuff